conf/cassandra/migration/default/V3__query_alerts.cql

```diff
@@ -0,0 +1,62 @@
+-- description: update to new query-based alert model
```

Comment:
This seems like its begging for a major version bump and a new tablespace or naming to allow the install to happen cleanly.